media/css/base/mozilla-modal.less

```diff
@@ -70,6 +70,7 @@ html.noscroll body {
                     .image-replaced();
                     border-radius: 50%;
                     border: 2px solid #fff;
+                    transition: transform 0.15s ease;
```

Comment:
Nit: use `.transition()` mixin? It makes no difference on this page, but other's could benefit from it?